server/controllers/review.js

```diff
@@ -0,0 +1,34 @@
+const Review = require("../models/Review");
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+exports.getReviews = asyncHandler(async (req, res) => {
+    const { id } = req.user;
+    let reviews = await Review.find({ profileId: id });
+    res.status(200).json({ success: { reviews }});
+});
+exports.createReview = asyncHandler(async (req, res) => {
+    const {
+        reviewerId,
+        profileId,
+        star,
+        description
+    } = req.body;
+    let review = new Review({
+        reviewerId,
```

Comment:
We don't need to send the `reviewerId` using the request body, why don't we access it using the request object instead - `req.user.id`

Reply:
changed to using req.user.id